### DISCUSSION TITLE
lsd and pcloud

### DISCUSSION THREAD

**Discussion by stkania** [lsd-rs/lsd#955] [General]
Hi,
I use lsd on my local filesystem without any problem, but as soon as I change into my pcloud-folder I get:

cannot open path of the current working directory: Permission denied

How can I get lsd working in my cloud folder?

**Comment by zwpaper** [comment]
we introduced a flag `--permission disable` on the master, it may worth a try, but not yet released

**Comment by stkania** [comment]
Thank's but I think my version of lds is to old. I installed via snap on ubuntu and lsd --version is showing "lsd 0.16.0". I tried it with the snap-version to first test lsd ;-). But now I would like to install a newer version. Maybe it's working there, the old version is not knowing  --permission.

**Reply by zwpaper** [reply]
snap has been reported with some problems like those, so it not maintained anymore, please try another way, maybe download a binary would be a good choice
